app/Validators/Establishment/Create.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+// eslint-disable-next-line
+const { ValidationException } = use('@adonisjs/validator/src/Exceptions')
+
+class Create {
```

Comment:
Should be named "Store" to match adonis naming convention
Also, should have a "Update" validator

Reply:
Resolved by 52d5a27